content/general/aws-authentication.textile

```diff
@@ -0,0 +1,267 @@
+---
+title: AWS authentication
+section: general
+index: 20
+meta_description: "There are two AWS authentication schemes that can be used when working with Ably: Credentials and the ARN of an assumable role."
+meta_keywords: "Ably, AWS, credentials, ARN, ARN of assumable role, SQS, Lambda, Kinesis."
+jump_to:
+  Help with:
+    - Credentials#aws-credentials
+    - ARN of an assumable role#aws-arn
+    - Create a policy#create-policy
+    - Create a role#create-role
+    - Testing the rule#testing-rule
+---
+
+When adding an Integration rule for an AWS endpoint such as for an "AWS Lambda function rule":/general/events/aws-lambda/, or a "Firehose rule":/general/firehose/ for AWS Kinesis or AWS SQS, there are two AWS authentication methods that can be used with Ably:
+
+1. Credentials
+2. ARN of an assumable role 
+
+h2(#aws-credentials). Credentials
+
+These are a set of credentials for an AWS IAM user that has permission to invoke your Lamda function, and, in the case of a Firehose rule, publish to your AWS SQS queue or AWS Kinesis stream. These credentials comprise of the 'access key id' and the 'secret access key' for the AWS IAM user. These are entered into the rule dialog as @access_key_id:secret_access_key@, that is, as a key-value pair, joined by a single colon (without a space). You can read more about these credentials in the AWS blog article "How to quickly find and update your access keys, password, and MFA setting using the AWS Management Console":https://aws.amazon.com/blogs/security/how-to-find-update-access-keys-password-mfa-aws-management-console/.
```

Comment:
```suggestion
These are a set of credentials for an AWS IAM user that has permission to invoke your Lambda function, and, in the case of a Firehose rule, publish to your AWS SQS queue or AWS Kinesis stream. These credentials comprise of the 'access key id' and the 'secret access key' for the AWS IAM user. These are entered into the rule dialog as @access_key_id:secret_access_key@, that is, as a key-value pair, joined by a single colon (without a space). You can read more about these credentials in the AWS blog article "How to quickly find and update your access keys, password, and MFA setting using the AWS Management Console":https://aws.amazon.com/blogs/security/how-to-find-update-access-keys-password-mfa-aws-management-console/.
```

Reply:
"These credentials comprise of the"

This should be "comprise the" or "consist of the", but not "comprise _of_ the"